ui/css/design-system/attributes.scss

```diff
@@ -50,36 +33,20 @@ $fractions: (
   8\/12: 66.666667%,
   9\/12: 75%,
   10\/12: 83.333333%,
-  11\/12: 91.666667%,
+  11\/12: 91.666667%
 );
 
-$sizes-numeric:
-  0,
-  1,
-  2,
-  3,
-  4,
-  5,
-  6,
-  7,
-  8,
-  9,
-  10,
-  11,
-  12;
+$sizes-numeric: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12;
 
-$sizes-strings:
-  xs,
-  sm,
-  md,
-  lg,
-  xl,
-  none;
+$sizes-strings: xs, sm, md, lg, xl, none;
 
 $border-style: solid, double, none, dashed, dotted;
 $directions: top, right, bottom, left;
 $display: block, grid, flex, inline-block, inline-grid, inline-flex, list-item;
 $text-align: left, right, center, justify, end;
 $font-weight: bold, normal, 100, 200, 300, 400, 500, 600, 700, 800, 900;
 $font-style: normal, italic, oblique;
-$font-size: 10px, 12px;
+$font-size: 12px;
```

Comment:
why was this change required?

Reply:
Hmm apologies, I can't remember maybe just revert. This is being used on the Typography component with the `fontSize` prop but it probably shouldn't exist. That's what the `variant` prop is for... but that's a different discussion